bors.toml

```diff
@@ -0,0 +1,8 @@
+status = [
+    "Julia 1.0.5 - ubuntu-latest - x64",
+    "Julia 1.3 - ubuntu-latest - x64",
+    "Julia 1 - windows-latest - x64",
```

Comment:
```suggestion
    # "Julia 1 - windows-latest - x64",
```

Reply:
Ah good catch. I was basing all of this off of [JLSO.jl](https://github.com/invenia/JLSO.jl/blob/master/.github/workflows/CI.yml)'s CI setup.

I guess one day I'll look into enabling Windows CI, but today's not that day.